docs/development.md

```diff
@@ -3,35 +3,30 @@
 
 ## Getting the source
 
-Clone from gerrit:
+Fork the repository: Navigate to [imageio repository](https://github.com/oVirt/ovirt-imageio)
+and click "Fork" button in right upper corner of the page.
 
-    git clone git://gerrit.ovirt.org/ovirt-imageio
+Clone the repo from GitHub:
 
-Install the commit message hook:
+    git clone git@github.com:<your-username>/ovirt-imageio.git
 
-    wget -P .git/hooks https://gerrit.ovirt.org/tools/hooks/commit-msg
+Add [imageio repository](https://github.com/oVirt/ovirt-imageio) as an
+upstream repository:
 
-Edit the commit-msg hook to add Signed-off-by header. Add this at the
-end of the commit-msg hook:
+    git remote add upstream git@github.com:oVirt/ovirt-imageio.git
 
-    # Add Signed-off-by trailer.
-    sob=$(git var GIT_AUTHOR_IDENT | sed -n 's/^\(.*>\).*$/Signed-off-by: \1/p')
-    git interpret-trailers --in-place --trailer "$sob" "$1"
+Fetch upstream changes (if you clone right after forking the repo, there
+sholdn't be any):
 
-Make the commit-msg hook executable:
-
-    chmod +x .git/hooks/commit-msg
+    git fetch upstream
+    git merge upstream/master master
```

Comment:
If we use fetch, better use rebase:

    git fetch upstream
    git rebase upstream/master

I use a simpler version:

    git pull upstream master

And I make sure that master never have any change - it always rebased on 
upstream, and all changes are done in topic branches.